### PR TITLE
 Adding unit tests for AWSMutationDatabaseAdapterTests

### PR DESCRIPTION
**Pull request by wooj2**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginTests/Sync/AWSMutationDatabaseAdapterTests.swift [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    var databaseAdapter: AWSMutationDatabaseAdapter!
+
+    let model1 = Post(title: "model1", content: "content1")
+
+    override func setUp() {
+        do {
+            let mockStorageAdapter = MockSQLiteStorageEngineAdapter()
+            databaseAdapter = try AWSMutationDatabaseAdapter(storageAdapter: mockStorageAdapter)
+        } catch {
+            XCTFail("Failed to setup sut")
```
Comment (palpatim) [inline AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginTests/Sync/AWSMutationDatabaseAdapterTests.swift]: What's a 'sut'? :)

Comment (wooj2) [reply AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginTests/Sync/AWSMutationDatabaseAdapterTests.swift]: sut is system under test -- i can spell this out, or we can adopt sut

Comment (wooj2) [reply AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginTests/Sync/AWSMutationDatabaseAdapterTests.swift]: updated wording to `system under test`

Comment (palpatim) [reply AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginTests/Sync/AWSMutationDatabaseAdapterTests.swift]: Got it. I don't object to it, I just don't use the term regularly, so I didn't recognize it as an acronym without the all-caps spelling :)

**Pull request merged**